main.go

```diff
@@ -105,6 +105,9 @@ type Options struct {
 	// DNS cache maximum TTL value - overrides record value
 	CacheMaxTTL uint32 `long:"cache-max-ttl" description:"Maximum TTL value for DNS entries, in seconds."`
 
+	// If true, optimistic DNS cache is enabled
```

Comment:
I think it's better to add more information to this comment.  Something like:

> CacheOptimistic, if set to true, enables the optimistic DNS cache.  That means that cached results will be served even if their cache TTL has already expired.

Reply:
Yes it's much better!